app/views/shared/_comments.html.erb

```diff
@@ -0,0 +1,10 @@
+<br>
+<ul class="comments">
+  <% if post.comments.empty? %>
+    <li>There are no comments yet.</li>
+  <% else %>
+    <%= post.comments.each do |comment| %>
+      <%= render partial: 'shared/comment', locals: { comment: comment } %>
+    <% end %>
+  <% end %>
+</ul>
```

Comment:
There are a couple of issues with this view: 
- The list of comment objects are being rendered after the list of comments that are properly listed: 
![image](https://github.com/ezraermy/rails-blog/assets/93520178/ea04d54e-f326-42e8-aa94-ceeaf67bce5a)
- The list of comments in the user's `show` view should only list the last 5 comments added to the post ( the result of the recent_comments method in the Post model ). As you can see in the previous image, 8 comments are being shown. 
- The number of comments at the top is not being updated.